.github/workflows/build-common.yml

```
@@ -267,6 +267,14 @@ jobs:
           # gradle enterprise is used for the build cache
           gradle-home-cache-excludes: caches/build-cache-1
 
+      - id: read-build-scan
+        name: Read build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo "build-scan=$(cat build-scan.txt)" >> $GITHUB_OUTPUT
+      - name: Build scan
+        if: ${{ hashFiles('build-scan.txt') != '' }}
+        run: echo ${{ steps.read-build-scan.outputs.build-scan }}
```

I think this should work, I'll accept the suggestion and see...

```suggestion
      - name: Build scan
        if: ${{ hashFiles('build-scan.txt') != '' }}
        run: cat build-scan.txt
```